client/src/components/common/NavTab/Nav.jsx

```diff
@@ -0,0 +1,52 @@
+import { NavLink } from 'react-router-dom';
+
+import { Menu } from 'antd';
+
+import './Navbar.css';
+import 'antd/dist/antd.min.css';
+
+import { DownOutlined } from '@ant-design/icons';
+
+const { Item, SubMenu } = Menu;
+
+const NAV = () => (
+  <Menu style={{ width: '34%' }} mode="horizontal">
+    <Item key="alipay">
+      <NavLink to="/home">Home</NavLink>
+    </Item>
+    <SubMenu key="shop" title="Shop" icon={<DownOutlined />}>
+      <Menu.ItemGroup>
+        <Menu.Item key="setting:1">shop 1</Menu.Item>
```

Comment:
You should destructure `ItemGroup` and you already destructured `Item` 
```suggestion
      <ItemGroup>
        <Item key="setting:1">shop 1</Item>
```